Usiqee/Module/News/Cell/NewsCarouselItemCell/NewsCarouselItemCell.swift

```diff
@@ -55,6 +55,7 @@ class NewsCarouselItemCell: UICollectionViewCell {
         authorView.isHidden = false
         let authorStorage = Storage.storage().reference(forURL: author.avatar)
         authorAvatar.sd_setImage(with: authorStorage)
+        authorAvatar.layer.cornerRadius = authorAvatar.frame.width/2
```

Comment:
is there a need for that? normally when inheriting from  `CircularImageView`, this already done